main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_info = json.load(players_file)
+    for player_name, player_info in players_info.items():
+        race_name = player_info["race"]["name"]
+        Race.objects.get_or_create(
+            name=race_name,
+            defaults={
+                "description": player_info["race"]["description"]
+            }
+        )
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                defaults={
+                    "bonus": skill["bonus"],
+                    "race": Race.objects.get(name=race_name)
+                }
+            )
+        if isinstance(player_info["guild"], dict):
+            guild_name = player_info["guild"]["name"]
+            Guild.objects.get_or_create(
+                name=guild_name,
+                defaults={
+                    "description": player_info["guild"]["description"]
+                }
+            )
+        player_guild = Guild.objects.get(
+            name=player_info["guild"]["name"]
+        ) if isinstance(player_info["guild"], dict) else None
```

Comment:
As for me strange, what about you?
Why you can not do before if isinstance(player_info["guild"], dict):
player_guild = None
and then inside if when you do get_or_create player_guild, created = Guild.objects.get_or_create